.github/workflows/tests.yml

```diff
@@ -53,6 +53,9 @@ jobs:
       - name: Checkout ${{ github.sha	}}
         uses: actions/checkout@v2
 
+      - name: Ensure tags exist for container packaging
+        run: git fetch --prune --unshallow --tags
```

Comment:
Without this are not you not able to do `git describe --tags --abbrev=0` to get the latest tag number? I've done that in other projects with Github Actions and it worked just fine.

Reply:
Yea it's why [this one initially failed](https://github.com/OSC/ondemand/actions/runs/451449535). 

It seems to be running these commands which merge against master and only have 1 commit in the `git log` at that point. I had to look at the action and run the commands myself to see what was going on. 
```
git clone git@github.com:OSC/ondemand.git
git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +55616edeb7114365b5d790eda041832f52ffd1ab:refs/remotes/pull/778/merge
```